src/rsqf.h

```diff
@@ -9,6 +9,8 @@ extern "C" {
 #include "constants.h"
 #include "remainder.h"
 
+typedef uint64_t elt_t;
+
 typedef struct rsqf_block_t {
   rem_t remainders[64];
   uint64_t occupieds;
```

Comment:
`occupieds` and `runends` are `uint64_t`. i'm not sure if they are `elt_t` so i left them as is.